src/Migrations/M011RemoveSmartsheetOption.php

```diff
@@ -18,8 +19,15 @@ class M011RemoveSmartsheetOption extends MigrationScript {
 	 * @return void
 	 */
 	protected static function execute( MigrationRecord $record ): void {
+		// Toggle between GoogleSheet and SmartSheet.
 		$options = get_option( Features::OPTIONS_KEY );
 		unset( $options['google_sheet_replaces_smartsheet'] );
 		update_option( Features::OPTIONS_KEY, $options );
+
+		// Smartsheet ID and legacy toggle.
+		$options = get_option( Settings::KEY );
+		unset( $options['analytics_local_smartsheet_id'] );
+		unset( $options['google_sheet_replaces_smartsheet'] );
```

Comment:
Is `google_sheet_replaces_smartsheet` flag present in both Features::OPTIONS_KEY & Settings::KEY ?

Reply:
Yes, I was surprised too when I saw the values there; but Features were once part of the planet4_options, and when we moved them out we still left the features values in options, we even still have [some code that does a fallback on it](https://github.com/greenpeace/planet4-master-theme/commit/5efca4b3538a2b52d64f2d87b85230221134a0d6#diff-ecf1d8c1cedf861e4a52c13d299a24fc41ebadbecc85d321eacbf40b4a3b3799R200-R203)